src/index.html

```diff
@@ -6,9 +6,76 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a href="#" class="header__image">
+        <img src="images/logo.png" alt="|MOYO|">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item-first">
+            <a
+            href="#"
+            class="nav__link is-active ">
```

Comment:
when you have 2 attributes they should be in the same line, if line is too long every attribute should start from new line, you have something wrong here
```suggestion
<a href="#" class="nav__link is-active">apple</a>
//or
<a href="#" class="nav__link is-active">
    apple
</a>
```